dotcom-rendering/src/model/enhanceTreats.ts

```diff
@@ -0,0 +1,9 @@
+import type { FEFrontCard, TreatType } from '../types/front';
+
+export const enhanceTreats = (treats: FEFrontCard[]): TreatType[] =>
+	treats.map((treat) => {
+		return {
+			text: treat.header.headline,
+			linkTo: treat.properties.href ?? treat.header.url,
+		};
+	});
```

Comment:
Might as well return directly…

```suggestion
	treats.map((treat) => ({
		text: treat.header.headline,
		linkTo: treat.properties.href ?? treat.header.url,
	}));
```